pkgs/servers/mattermost/default.nix

```diff
@@ -34,7 +34,7 @@ buildGoModule rec {
 
   webapp = fetchurl {
     url = "https://releases.mattermost.com/${version}/mattermost-${version}-linux-amd64.tar.gz";
-    hash = "sha256-OVq+puaXPJGOmW9pcgX24LXBfYLzuczRY7KG2tQBY3Q=";
+    hash = "sha256-tgds8eTBeisuJcLgtx6zOiFUcVL1oU0LLbPqmh4SQUU=";
   };
 
   vendorHash = "sha256-TJCtgNf56A1U0EbV5gXjTro+YudVBRWiSZoBC3nJxnE=";
```

Comment:
The `vendorHash` has not changed with that update?

Reply:
it has not. this happens fairly regularly, actually — I test for it each time by changing it, then looking what the error / reported hash is.

(it occurs to me that having to check this is kinda underdocumented knowledge; I'm not even sure if things like `nix-update` would do this automatically — not that i use it much in the first place …)

Reply:
For some reason the nixpkgs update script for this didn't automatically work for this version, or the last few versions. This looks good to me, though. Weird. Any ideas why?

https://github.com/NixOS/nixpkgs/pull/312507/files#diff-77db8a1c51d8413b4676ee2bb54fe66fd1ee9a14bd98efe422b1b593b8d73b5dR72

Reply:
tbh i'm not sure it *could* work. Which is to say, it mysteriously fails every time rn, but even if it didn't does it more than `nix-update` would, which (having tried it) fails to update e.g. the frontend's hash?

So afaict even in the best case it would result in unfinished PRs with mistakes which are easy to miss during review.